# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openzone-cursors/install.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- openzone-cursors-1.2.9/Makefile.vanilla	2020-09-21 12:29:05.250701914 +0000
+++ openzone-cursors-1.2.9/Makefile	2020-09-21 12:32:21.190702007 +0000
@@ -16,7 +16,7 @@
 
 install: $(themes)
 	install -dm755 $(DESTDIR)/share/icons
-	mv $(themes) $(DESTDIR)/share/icons
+	for t in $(themes); do mkdir $(DESTDIR)/share/icons/$$t; cp -vrf $$t/* $(DESTDIR)/share/icons/$$t/; done
 
 pack: $(themes)
 	tar Jcf $(pkgname)-$(version).tar.xz --owner=0 --group=0 $(themes)
